controller-server/src/main/java/com/yahoo/vespa/hosted/controller/application/EndpointId.java

```diff
@@ -50,4 +53,10 @@ private static String requireNotEmpty(String input) {
     public static EndpointId default_() { return DEFAULT; }
 
     public static EndpointId of(String id) { return new EndpointId(id); }
```

Comment:
Since we have this static constructor, should we just make the regular constructor private as well?